### ISSUE TITLE
Feature request: message on no issues found

### ISSUE THREAD

**Issue by RoFz** [hadolint/hadolint#1006]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Hadolint to inform/acknowledge when no issues are found.

```bash
$ hadolint components/runner/Dockerfile 
No issues found with components/runner/Dockerfile
$
```

### Actual behavior

Hadolint does not produce any output when no issues are found.

```bash
$ hadolint components/runner/Dockerfile 

$
```

### Steps to reproduce the behavior

Execute hadolint against any Dockerfile.

```bash
$ hadolint --version
Haskell Dockerfile Linter 2.12.0

$ hadolint components/runner/Dockerfile 

$
```

This feature would be really useful since we're also using hadolint's GitHub Action along with _thollander/actions-comment-pull-request_ to supply scan results to our pull requests. Unfortunately, hadolint does not produce any output when no issues are found. It would be great to have something like what Aquasec's trivy does:

```
components/hello-world/output/hello-world.tar (alpine 3.19.1)
=============================================================
Total: 0 (HIGH: 0, CRITICAL: 0)
```
